File: javadoc-service/src/main/java/com/michibaum/javadoc_service/JavadocServiceApplication.java
package com.michibaum.javadoc_service;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.client.discovery.EnableDiscoveryClient;
import org.springframework.scheduling.annotation.EnableScheduling;

@SpringBootApplication
@EnableDiscoveryClient
@EnableScheduling
public class JavadocServiceApplication {

	public static void main(String[] args) {
		SpringApplication.run(JavadocServiceApplication.class, args);
	}

}

File: javadoc-service/src/main/kotlin/com/michibaum/javadoc_service/JavadocServiceApplication.kt
package com.michibaum.javadoc_service

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.cloud.client.discovery.EnableDiscoveryClient
import org.springframework.scheduling.annotation.EnableScheduling
import org.springframework.boot.runApplication

@SpringBootApplication
@EnableDiscoveryClient
@EnableScheduling
class JavadocServiceApplication

fun main(args: Array<String>) {
    runApplication<JavadocServiceApplication>(*args)
}